openedx/features/learner_profile/static/learner_profile/js/views/learner_profile_view.js

```diff
@@ -53,10 +53,15 @@
                         ownProfile: this.options.ownProfile
                     });
 
+
                     HtmlUtils.setHtml(this.$el, HtmlUtils.template(learnerProfileTemplate)({
                         username: self.options.accountSettingsModel.get('username'),
+                        name: self.options.accountSettingsModel.get('name'),
                         ownProfile: self.options.ownProfile,
-                        showFullProfile: self.showFullProfile()
+                        showFullProfile: self.showFullProfile(),
+                        profile_header: gettext('My Profile'),
+                        profile_subheader: gettext('Build out your profile to personalize your identity on '
+                            + self.options.platformName)
```

Comment:
This should use variable replacement: 'Build out your profile to personalize your identity on {platformName}'